core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -1572,9 +1572,13 @@ export class WorkflowEditorComponent implements AfterViewInit {
    * and converts that event to a workflow action
    */
   private handleLinkBreakpointButtonClick(): void {
-    fromEvent<JointPaperEvent>(this.getJointPaper(), "tool:breakpoint", {
-      passive: true,
-    })
+    fromEvent<JointPaperEvent>(
+      this.getJointPaper(),
+      "tool:breakpoint"
+      // , {
+      //   passive: true,
+      // }
```

Comment:
is this intentional?

Reply:
I just removed it. RxJS's new API doesn't allow this argument anymore, and I checked it seems this argument is not used. Removing it seems to have no effect.

Reply:
done